sample.html

```diff
@@ -143,6 +145,13 @@ <h1 class="form-name">Form Name</h1>
           throw new Error("Please enter a valid email")
         }
       })
+
+      validate("github", required("Github User Profile Link is required"), async value => {
+        const githubformat = /^[h][t][t][p](s?)[:][/][/]github\.com[/]|^github\.com[/]/
```

Comment:
```suggestion
        const githubformat = /^(?:https?:\/\/)?github\.com\/[a-zA-Z0-9_]+$/
```
This will work, try understanding how it works tho